boilerplate/app/screens/DemoCommunityScreen.tsx

```diff
@@ -13,9 +13,11 @@ const reactNativeNewsletterLogo = require("../../assets/images/demo/rnn-logo.png
 
 export const DemoCommunityScreen: FC<DemoTabScreenProps<"DemoCommunity">> =
   function DemoCommunityScreen(_props) {
+    const textRef = useRef<RNText | null>(null)
+
     return (
       <Screen preset="scroll" contentContainerStyle={$container} safeAreaEdges={["top"]}>
-        <Text preset="heading" tx="demoCommunityScreen.title" style={$title} />
+        <Text preset="heading" tx="demoCommunityScreen.title" style={$title} ref={textRef} />
```

Comment:
I think if we're not using the ref for anything, we don't need to add it to the demo here.